✨(captureimage) add a video loopback of the camera chosen for image p…

juljac56 opened the pull request:
Worked on file CaptureImage.tsx and CaptureImage.css

The goal was to add a video loopback of the camera selected by the user, used for sending images of the whiteboard. It is useful as these feature allows the user set up the camera at the right position.
By default, when a camera is selected, the loopback is displayed. But it is possible to hide this loopback video by clicking on the icon "double arrow".

juljac56 committed:
✨(captureimage) add a video loopback of the camera chosen for image processing

add a video loopback for the camera selected by the user for sending images of the whiteboard, it is useful to allow the user set up the camera at the right position.

AladdinVR commented on box-ui/src/components/ts/CaptureImage.tsx:
```diff
@@ -104,65 +148,85 @@ const CaptureImage: FunctionComponent<CaptureImageProps> = (props: CaptureImageP
-                    ]}
-                    setDisplayFocus={setDisplayFocus}
-                />
+            {false && (
+                <div className='arrowContainer' onClick={changeViewLoopback}>
+                    <DoubleArrowIcon className={showLoopback ? 'arrowDown' : 'arrowUp'} />
+                </div>
```
never shown

djexvr commented on box-ui/src/components/ts/CaptureImage.tsx:
```diff
@@ -9,13 +9,18 @@ import '../css/CaptureImage.css';
     const [cameraList, setCameraList] = useState<MediaDeviceInfo[]>([]);
     const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
     const [displayFocus, setDisplayFocus] = useState(false);
+    const [videoLoopback, setVideoLoopback] = useState<MediaStream>();
```
You don't use this state

juljac56 committed:
🔥(CaptureImage) take into account the review

remove useless state and a div never displayed

juljac56 committed:
🔀(captureimage) merge branch videoloopback with main

resolve conflicts in capture image component

juljac56 committed:
✨(captureimage) add recording icon outside of the loopback video container

add the red recording icon outside of the loopback video in order to indicate to the user if a camera is selected, even if the loopback video is not displayed
